### DISCUSSION TITLE
DPGEN encountered the following issue while exploring the potential function of the PtNiMoO system.

### DISCUSSION THREAD

**Discussion by wangsl613** [deepmodeling/dpgen#1966] [Q&A]
### Summary

DPGEN cannot perform the exploration step effectively.

### DeePMD-kit Version

2.2.10

### Backend and its version

tensorflow                2.16.1；dpdata                    0.2.7；dpgen                     0.10.6；

### Python Version, CUDA Version, GCC Version, LAMMPS Version, etc

python                    3.8.0；lammps                    2023.08.02；gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-39)；CentOS Linux release 7.8.2003 (Core)

### Details

Dear DeePMD developers,

I used the same initial structure to run model_devi, only increasing the temperature by 100K. However, all the structures from the model_devi runs failed. Could it be that the temperature increase is too high, or is there an issue with my init_data (300K, ran over 800 steps of AIMD)? All structures are from the PtNiMoO system. Additionally, the 152 candidates from the first iteration's fp calculations resulted in similar structures.

Best regards.
![2568be51ea55ae0ea0fa95f0ed7d916](https://github.com/user-attachments/assets/80c8c8f2-fcc6-4c12-ae61-5846ca0fcd95)

![08674542f9eb37cedba2420b258556d](https://github.com/user-attachments/assets/e8bb87a5-05ab-4338-b3dc-ed8df79e84df)

[dpgen.tar.gz](https://github.com/user-attachments/files/16407459/dpgen.tar.gz)

init.tar.gz：链接: https://pan.baidu.com/s/14jdGMoeSeYAd_7rdsgn5lA?pwd=w6ni 提取码: w6ni

**Comment by njzjz** [comment]
Do you get a different result when performing simulations under 100K? The current result might be expected, as structures under 200K are very different from that under 100K.

**Comment by wangsl613** [comment]
Thanks for the reply but my dataset is from AIMD at 300K temperature conditions.

[a comment by an automated account is not shown]
